README.md

```diff
@@ -67,6 +67,53 @@ This Solidity project uses Foundry as a toolkit. If you don't have Foundry insta
     forge snapshot
     ```
 
+### Generating Code Coverage Report
```

Comment:
for the readme, `forge coverage` works out of the box, and so I think this more detailed description is too much for the readme. I ll set it up in a script and we can run a hosted version of the coverage report and then just link the results to the readme